src/lean_spec/subspecs/containers/block/block.py

```diff
@@ -56,14 +57,29 @@ class Block(Container):
     """The block's payload."""
 
 
-class SignedBlock(Container):
-    """A container for a block and the proposer's signature."""
+class BlockAndProposerVote(Container):
```

Comment:
should we go with ProposerVote or ProposerAttestation nomenclature here @syjn99 @unnawut